shimmingtoolbox/cli/check_env.py

```diff
@@ -83,7 +83,10 @@ def check_prelude_installation():
         bool: True if prelude is installed, False if not.
     """
     try:
-        subprocess.check_call(['which', 'prelude'], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+        if sys.platform != 'win32':
+            subprocess.check_call(['which', 'prelude'], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+        else:
+            subprocess.check_call(['where', 'prelude'], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```

Comment:
Would switch the if to == 'win32', else

Reply:
Done here: 18c7d78d82f45626ff350eda5b758557ffb63187